internal/datastore/crdb/tx.go

```diff
@@ -67,7 +69,7 @@ func executeWithResets(ctx context.Context, fn innerFunc, maxRetries uint8) (err
 	}
 
 	// The last error was resettable but we're out of retries
-	return fmt.Errorf(errReachedMaxAttempts, retries, maxRetries+1, err)
+	return fmt.Errorf(errReachedMaxRetries, retries, maxRetries+1, err)
 }
 
 func resettable(ctx context.Context, err error) bool {
```

Comment:
Can we add some sort of unit test for this method?

Reply:
@josephschorr the method seems fairly tested indirectly [through table tests invoking](https://github.com/authzed/spicedb/blob/ad65811afa4efeb2ca157cb6a876141163f57f28/internal/datastore/crdb/tx_test.go#L43) `executeWithResets`. Just run code coverage and showed  100% code coverage.

With that said, I can still add unit tests if you feel strongly about it 👍🏻

Reply:
Nope, that's fine